tests/xdp/xdp_tests.h

```diff
@@ -2,5 +2,14 @@
 // SPDX-License-Identifier: MIT
 
 #pragma once
+#include <netiodef.h>
 #include <winsock2.h>
 #include <ws2tcpip.h>
+
+struct _udp_header
```

Comment:
tests\sample\ebpf.h defines UDP_HEADER already, can you just use that?

Reply:
The test app includes netiodef which has definitions for ethernet and IP headers but not udp header. Replacing netiodef with ebpf.h in xdp_tests.h causes linker issues with ntohs definition which I was not able to resolve. Is this a blocking comment?

Reply:
We should not have 2 definitions in the same repo, it would be fine to move UDP_HEADER out of ebpf.h and into something like udp.h that can be included by both.

Reply:
created a new ebpf_udp.h.